test/input_test.py

```diff
@@ -1,28 +1,35 @@
 from unittest import TestCase
-from passflip.input import create_prompt_message
+import passflip.input
 
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 class InputPromptMessageTest(TestCase):
```

Comment:
These tests have changed quite a bit and the class name no longer reflects their intent. Maybe `PromptTest` or something would suit better.